src/lib/y2network/widgets/wireless_tab.rb

```diff
@@ -36,20 +38,68 @@ def initialize(builder)
       end
 
       def label
-        _("&Wireless Specific")
+        _("&Wireless")
       end
 
       def contents
         VBox(
-          VSpacing(0.5),
```

Comment:
These "0.5" spacing used to have a reason due to different handling in ncurses and qt. No idea whether it is still relevant

Reply:
Yes, that's true. But the original UI was crowded. Now, with the new layout, we can put some additional space here :smiley:

Reply:
Well, not so much if we want to fit well with 80x24 in ncurses. If we use more space then EAP configuration is a problem as it uses the whole interface.

Reply:
Yes, but it still fits because we removed a frame and a field. In the future, we could reconsider this change.